gui/wxpython/vdigit/wxdisplay.py

```diff
@@ -712,7 +712,7 @@ def SelectLineByPoint(self, point, ltype=None, poMapInfo=None):
         pz = c_double()
         if not self._validLine(lineNearest):
             return {"line": -1, "point": None}
-        ftype = Vect_read_line(poMapInfo, self.poPoints, self.poCats, lineNearest)
+        _ = Vect_read_line(poMapInfo, self.poPoints, self.poCats, lineNearest)
```

Comment:
I think there might be an issue here, as the gettext translation function `_()` is using the underscore name, and here it is assigning something else. I would expect that everything calling a translation after this line to be broken (at least in that file).

Reply:
If the function return value is truly never needed and the function is called only for its side effects (filling self.poPoints and self.poCats), I have called it without assignment